pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -727,34 +732,38 @@ IS 'returns the series id that exactly matches a JSONB of labels';
 GRANT EXECUTE ON FUNCTION SCHEMA_PROM.series_id(jsonb) TO prom_writer;
 
 CREATE OR REPLACE  FUNCTION SCHEMA_CATALOG.get_series_id_for_key_value_array(metric_name TEXT, label_keys text[], label_values text[])
```

Comment:
This is also improperly named according to our naming convention as it also creates. I would also be okay with renaming all the get or create functions to goc_ or gc_ or something if we think that that would be better for name length etc.